src/components/Calculator.js

```diff
@@ -1,47 +1,73 @@
-              <button type="button" className="five itemcalc heightbtn">5</button>
-              <button type="button" className="six itemcalc heightbtn">6</button>
-              <button type="button" className="one itemcalc heightbtn">1</button>
-              <button type="button" className="two itemcalc heightbtn">2</button>
-              <button type="button" className="three itemcalc heightbtn">3</button>
+              <button type="button" onClick={this.handleClick} className="ac itemcalc heightbtn" value="AC">AC</button>
+              <button type="button" onClick={this.handleClick} className="addneg itemcalc heightbtn" value="+/-">+/-</button>
+              <button type="button" onClick={this.handleClick} className="percent itemcalc heightbtn" value="%">%</button>
+              <button type="button" onClick={this.handleClick} className="seven itemcalc heightbtn" value="7">7</button>
+              <button type="button" onClick={this.handleClick} className="eight itemcalc heightbtn" value="8">8</button>
+              <button type="button" onClick={this.handleClick} className="nine itemcalc heightbtn" value="9">9</button>
+              <button type="button" onClick={this.handleClick} className="four itemcalc heightbtn" value="4">4</button>
+              <button type="button" onClick={this.handleClick} className="five itemcalc heightbtn" value="5">5</button>
+              <button type="button" onClick={this.handleClick} className="six itemcalc heightbtn" value="6">6</button>
+              <button type="button" onClick={this.handleClick} className="one itemcalc heightbtn" value="1">1</button>
+              <button type="button" onClick={this.handleClick} className="two itemcalc heightbtn" value="2">2</button>
+              <button type="button" onClick={this.handleClick} className="three itemcalc heightbtn" value="3">3</button>
             </div>
 
             <div className="calcNumber2">
-              <button type="button" className="zero itemcalc1 heightbtn">0</button>
-              <button type="button" className="dot itemcalc1 heightbtn">.</button>
+              <button type="button" onClick={this.handleClick} className="zero itemcalc1 heightbtn" value="0">0</button>
+              <button type="button" onClick={this.handleClick} className="dot itemcalc1 heightbtn" value=".">.</button>
             </div>
 
           </div>
 
           <div className="calcOperators">
-            <button type="button" className="divid">÷</button>
-            <button type="button" className="times">&times;</button>
-            <button type="button" className="minus">-</button>
-            <button type="button" className="add">+</button>
-            <button type="button" className="equal">=</button>
+            <button type="button" onClick={this.handleClick} className="divid" value="÷">÷</button>
+            <button type="button" onClick={this.handleClick} className="times" value="&times;">&times;</button>
```

Comment:
- [ ] The operator to multiply is not working as it should because you made the value `&times;` instead of `x`, the inner text can be &times;, but the value should be `x` so it could match what was provided in the operate.js file. Kindly change it.

Reply:
@phelian23  thank you